Admin/Admin.php

```diff
@@ -897,6 +904,11 @@ public function buildDatagrid()
         foreach ($this->getExtensions() as $extension) {
             $extension->configureDatagridFilters($mapper);
         }
+
+        // set template for SimplePager if it's not already overloaded
+        if ($this->getPagerType() === 'simple' && $this->templates['pager_results'] === 'SonataAdminBundle:Pager:results.html.twig') {
```

Comment:
On second thought I will try to define this using the dependency injection as the prefered way to changing pager type.